support-frontend/assets/components/page/offerStrapline.tsx

```diff
@@ -0,0 +1,100 @@
+import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
+import type { ReactElement, ReactNode } from 'react';
+import { offerStraplineBlue } from 'stylesheets/emotion/colours';
+
+const offerStraplineStyles = css`
+	font-family: GuardianTextSans, 'Helvetica Neue', Helvetica, Arial,
```

Comment:
you should be able to get the font from source:
`import { textSans } from '@guardian/source-foundations';`

Reply:
I spent ages trying to find out how to do that - in the end I copied what others had done in the codebase elsewhere.

Will definitely fix!

Reply:
Done